src/ngx_http_lua_worker.c

```diff
@@ -57,8 +84,25 @@ ngx_http_lua_ffi_worker_pid(void)
 
 
 int
+ngx_http_lua_ffi_worker_id(void)
```

Comment:
Because you're adding FFI C API here, you're going to create a PR for lua-resty-core as well? ;)